### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by shraiyya**
## Related Issuse

- Info about Issue or bug

Closes: #11
#### Describe the changes you've made

- ML algorithms used:
1. Logistic Regression
2. Naive Bayes
3. Support Vector Machine
4. K-Nearest Neighbours
5. Decision Tree
6. Random Forest
7. XGBoost
8. Artificial Neural Network using Keras
-  Accuracy achieved: **89%**
- Dataset used: _https://www.kaggle.com/ronitf/heart-disease-uci_
- Please add GSSOC Tag and level
## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by shubhigupta991** [comment]
you didn't added the model. Add that.Other things LGTM👍🏻

**Review by utkarsh0702 — CHANGES_REQUESTED** [review]
@shraiyya The visualizations are great.
A few points to improve:
1) Use a different kernel in SVM.
2) Try turning decision tree  and random forest with no. of estimators and max depth.
3) In neural network, add a few more hidden layers, increase each dropout to 0.3 atleast and if needed use regularizers during overfitting.
4) Store the regressor with the highest accuracy in pickle file format and provide us with the model.

**Comment by shraiyya** [comment]
Yes Sure I will work on this.

**Comment by shubhigupta991** [comment]
Any updates?

**Comment by utkarsh0702** [comment]
@shraiyya Please provide an update to the pr.

**Comment by shraiyya** [comment]
Hello !
I have been a bit busy due to college exams and projects. I don't think I will be able to complete this task . Sorry for the inconvenience caused.
Thanks again.

**Comment by kritikaparmar-programmer** [comment]
Ok @shraiyya closing this pr then :)

**Comment by kritikaparmar-programmer** [comment]
Hey @utkarsh0702 create a new issue for the changes asked by you. 
I'm merging this pr

**Review by kritikaparmar-programmer — APPROVED** [review]
